docs/glossary.rst

```diff
@@ -9,11 +9,13 @@ Flang glossary
 +======+============================================================================================+
 | ADT  | Abstract Data Types.                                                                       |
 +------+--------------------------------------------------------------------------------------------+
+| ASD  | Array Subscript Descriptor - holds the number of dimensions for an array, see ``ast.h.``   |
++------+--------------------------------------------------------------------------------------------+
 | AST  | Abstract Syntax Tree.                                                                      |
 +------+--------------------------------------------------------------------------------------------+
-| BE   | (used heavily in code and comments, short for "backend"                                    |
+| BE   | (used heavily in code and comments), short for "backend"                                   |
```

Comment:
Nit: There is no need for a comma after a parenthesized clause, and I would put the main definition first, e.g.
```
short for "backend" (used heavily in code and comments)
```

Reply:
The same comment applies to all rows below that have parenthetical clauses.